regression/cbmc/Pointer_byte_extract5/test.desc

```diff
@@ -4,7 +4,7 @@ main.i
 ^EXIT=10$
 ^SIGNAL=0$
 array\.List dynamic object upper bound in p->List\[2\]: FAILURE
-\*\* 1 of 11 failed 
+\*\* 1 of 12 failed
```

Comment:
:question: Why does this change? 
:pick: Probably best to set to `\d+` since number of goals isn't what is being tested

Reply:
Explained and changed.